test/functional/pdo_sqlsrv/pdo_538_quote_nul.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+Test the PDO::quote() method.
+--SKIPIF--
+<?php require('skipif.inc'); ?>
+--FILE--
+<?php
+  
+require_once 'MsCommon.inc';
+
+try 
+{      
+    $connection = connect(); 
+    //$connection->setAttribute( PDO::SQLSRV_ATTR_DIRECT_QUERY, PDO::SQLSRV_ENCODING_SYSTEM );
+
```

Comment:
Why did you comment this line? Have you tested this with PDO::SQLSRV_ENCODING_SYSTEM in Windows?

Reply:
I tested both ENCODING_UTF8 and ENCODING_SYSTEM and they are both OK no. I'll delete this line from the test.